cmd/ovpm/action_vpn.go

```diff
@@ -57,7 +57,7 @@ func vpnStatusAction(rpcServURLStr string) error {
 	return nil
 }
 
-func vpnInitAction(rpcServURLStr string, hostname string, port string, proto pb.VPNProto, netCIDR string, dnsAddr string) error {
+func vpnInitAction(rpcServURLStr string, hostname string, port string, proto pb.VPNProto, netCIDR string, dnsAddr string, keepalivePeriod string, keepaliveTimeout string, useLZO bool) error {
```

Comment:
too much arguments, pls replace with struct

Reply:
Yep, I know, but this is an initial application style. It will cause the API rewrite. If the author will agree, I will rewrite it in the next PRs.

Reply:
that's not API.. that's why I propose to replace so much arguments with one struct